src/main/java/uk/gov/cslearning/catalogue/api/SearchController.java

```diff
@@ -7,56 +7,36 @@
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
-import uk.gov.cslearning.catalogue.domain.Course;
-import uk.gov.cslearning.catalogue.domain.Resource;
 import uk.gov.cslearning.catalogue.domain.SearchPage;
+import uk.gov.cslearning.catalogue.domain.Status;
 import uk.gov.cslearning.catalogue.repository.CourseRepository;
-import uk.gov.cslearning.catalogue.repository.ResourceRepository;
 
-import java.util.ArrayList;
-
-import static com.google.common.base.Preconditions.checkArgument;
+import java.util.Arrays;
+import java.util.stream.Collectors;
 
 @RestController
 @RequestMapping("/search")
 public class SearchController {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SearchController.class);
 
-    private ResourceRepository resourceRepository;
-
     private CourseRepository courseRepository;
 
     @Autowired
-    public SearchController(ResourceRepository resourceRepository, CourseRepository courseRepository) {
-        checkArgument(resourceRepository != null);
-        this.resourceRepository = resourceRepository;
+    public SearchController(CourseRepository courseRepository) {
         this.courseRepository = courseRepository;
     }
 
-    @GetMapping
-    public ResponseEntity<SearchResults> search(String query, FilterParameters filterParameters, PageParameters pageParameters) {
-        LOGGER.debug("Searching resources with query {}", query);
+    @GetMapping("/courses")
+    public ResponseEntity<SearchResults> search(@RequestParam(name = "status", defaultValue = "Published") String status, String query, FilterParameters filterParameters, PageParameters pageParameters) {
+        LOGGER.debug("Searching courses with query {}", query);
 
         Pageable pageable = pageParameters.getPageRequest();
-        SearchPage searchPage = resourceRepository.search(query, pageable, filterParameters);
+        SearchPage searchPage;
+        searchPage = courseRepository.search(query, pageable, filterParameters, Arrays.stream(status.split(",")).map(Status::forValue).collect(Collectors.toList()));
```

Comment:
This would probably look cleaner as just 
`SearchPage searchPage = courseRepository.search(query, pageable, filterParameters, Arrays.stream(status.split(",")).map(Status::forValue).collect(Collectors.toList()));`